package.json

```diff
@@ -77,6 +78,7 @@
     "pg": "^8.9.0",
     "pg-query-stream": "^4.3.0",
     "pino": "^8.10.0",
+    "rate-limiter-flexible": "^2.4.1",
```

Comment:
Dependency of the primary package we use. Without this, build throws `ERROR: MODULE rate-limiter-flexible NOT FOUND`

Another option for rate limiting package is this: https://github.com/teamplanes/graphql-rate-limit

Though it doesn't seem to be actively maintained, last update was 2 years ago.